### ISSUE TITLE
Ability to set default scatter edge parameters in matplotlibrc

### ISSUE THREAD

**Issue by brewstronomy** [matplotlib/matplotlib#11933]
Issue #6272 brought up the idea to edit the matplotlibrc to set the default marker when calling plt.scatter(). Can this be updated to include other default parameters, e.g. linewidths and/or edgecolors?

**Comment by timhoffm** [comment]
Most parameters are actually controlled via rcParams already, e.g. `rcParams['lines.linewidth']`. The case of marker was a bit different, because it had a hard-coded value.

I'm not quite sure if we want to do per-plot-type settings of all the style parameters. It adds a bit more flexibility for default styles. On the other hand the styles would have to define many more parameters `lines.linewidth`, `scatter.linewidth`, ..., which most of the time will be identical. Or alternatively we would need to implement a cascaded style system with a general line width, which may be overriden with other per-plot-type settings.

If we decide that per-plot-type settings are desireable, I propose to develop such a cascaded style system first to keep the complexity under control. Personally, I'm -0.5 on this because the gain is not large enough compared to the amount of required work.

Another issue for colors (including edgecolors): The code is already quite complex and I strongly advise against adding more stuff in the current state. #11663 starts the color parsing refactoring and should be merged before changing anything color-related in `scatter()`.
